### ISSUE TITLE
[Fix] Fix the alignment of snake game board

### ISSUE THREAD

**Issue by Nitesh-thapliyal** [Bauddhik-Geeks/Snake-Game#4]

**Comment by ampsteric** [comment]
hey! , would love to work on this issue , please assign it to me.

**Comment by adityamangal1** [comment]
Hey, @ampsteric Go ahead @Nitesh-thapliyal will assign you soon. Meanwhile make sure to join the organization and discord server first so that your contribution is counted as hacktoberfest accepted.😊

Organization link👉 https://github.com/Bauddhik-Geeks/Welcome-to-Bauddhik-Geeks
Discord server link 👉 https://join.bauddhikgeeks.tech

**Comment by ampsteric** [comment]
Yeah I have joined! , please assign me this issue so that I can start working on it.
